packages/modules/src/modules/header/Header.tsx

```diff
@@ -69,7 +69,7 @@ const Header: React.FC<HeaderRenderProps> = (props: HeaderRenderProps) => {
 
             {primaryCta && (
                 <ThemeProvider theme={buttonReaderRevenueBrand}>
-                    <Hide below="mobileLandscape">
+                    <Hide below="tablet">
```

Comment:
we only display the copy above the cta from tablet, so this change makes rendering of the cta consistent with that